frontend/src/components/Metrics/CustomMetrics.tsx

```diff
@@ -198,6 +217,19 @@ class CustomMetrics extends React.Component<Props, MetricsState> {
     }
   }
 
+  renderFetchMetrics = (title, msg) => {
+    return (
+      <div className={emptyStyle}>
+        <EmptyState variant={EmptyStateVariant.small}>
+          <Title headingLevel="h5" size={TitleSizes.lg}>
+            {title}
+          </Title>
+          <EmptyStateBody>{msg}</EmptyStateBody>
```

Comment:
imo you don't need the message. Just "Loading metrics" is good enough and there's no guarantee that the metrics will load in a few seconds :). The graph page just has "Loading Graph" so it'd make it more consistent.

Reply:
> there's no guarantee that the metrics will load in a few seconds :)

I agree. Let's not give the user any kind of guarantee or even expectation of load times. What is a few seconds? 2 seconds? 5? 10? 30? 120? :) I would say just tell them its Loading and that's it.

Reply:
I first had the title only, but visually it seemed strange to have only Header without body. But agree that "few seconds" is odd.